source/octod/media.d

```diff
@@ -12,6 +12,23 @@
 module octod.media;
 
 /**
+    Enumerates some of commonly used formats
+ **/
+ enum MediaFormat
+ {
+     JSON = "json",
+     XML  = "xml",
+ }
```

Comment:
Params (raw, html, sha, etc.) missing. But I won't block this because of that anymore, we can do it in another PR.